doc/nrf/conf.py

```diff
@@ -183,6 +184,11 @@
 ncs_cache_config = NRF_BASE / "doc" / "cache.yml"
 ncs_cache_manifest = NRF_BASE / "west.yml"
 
+# Options for sphinx_notfound_page ---------------------------------------------
+
+notfound_urls_prefix = "/nRF_Connect_SDK/doc/{}/nrf/".format(
```

Comment:
Not gonna block but I wonder why you don't systematically use f-strings everywhere.

Reply:
I found it more readable this way here because there's logic (if/else)